### ISSUE TITLE
Implement individual-level line plots (phase 1)

### ISSUE THREAD

**Issue by danicahelb** [VEuPathDB/web-eda#1528]
Related ticket: https://github.com/VEuPathDB/web-eda/issues/1440

**Phase 1 must haves:**
- X-axis parameter selector: constrain to time-related variables (date, age, study timepoint)...these could be continuous or discrete (ie, 5 study timepoints). possibly think about defining this parameter on a per-study basis (in the presenter file? or with annotations in the .owl?)
- Y-axis parameter selector: constrain to continuous variables (we decided to handle the categorical vars in a separate strip plot viz tool, see https://github.com/VEuPathDB/web-eda/issues/1527)

**Phase 1 nice to haves:**
- Stratification identifier (ie, facet plots by PID vs HHID, etc)
- Sorting based on other variable(s) -- through faceting 2.0 (https://github.com/VEuPathDB/web-eda/issues/1501), which we anticipate rolling out BEFORE individual-level line plots

**Phase 2:**
- plot carpets, https://github.com/VEuPathDB/EdaNewIssues/issues/519

Do not start implementing until individual level line plot mockups receive final sign off in EDA UX meeting

**Comment by danicahelb** [comment]
Mockups are ready for EDA UX review: https://www.figma.com/file/SZwh2ZJnAG7ffRnR8tXZZs/Individual-Summaries?node-id=256%3A1145&t=VA44mZFtPAlDiKFI-0
